### PR TITLE
My solution 1 for level 0 programs

### PR DESCRIPTION
**Pull request by pranjalsharmahp**
Accept the pull request

### CODE REVIEW

**Comment by pranjalsharmahp** [comment]
Hello
Thank me

**Comment by tmklabs** [comment]
Good Boyyy

File: level0-solutions/solution1d.c [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    {
+        printf("You are entering a negative number \n");
+    }
+    else if (n==0)
+    {
+        printf("0 is not prime nor a composite number\n");
+    }
+    else
+    {
+        for (int i=2;i<=sqrt(n);i++)
```
Comment (tmklabs) [inline level0-solutions/solution1d.c]: Check this once

**Review by tmklabs — COMMENTED** [review]
Check it once

**Pull request merged**